### ISSUE TITLE
Output AuthenticationStateProvider to screen, not Console

### ISSUE THREAD

**Issue by bdnts** [dotnet/AspNetCore.Docs#16932]
Perform following changes
&lt;p&gt;@message&lt;/p&gt;
private string message {get; set; } = "None";
            message = $"{user.Identity.Name} is authenticated.";
            message = "The user is NOT authenticated.";

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c9942b0a-a7fb-0d34-9201-af0963affa9e
* Version Independent ID: 19770710-5225-a1ea-e082-65da6ba4a49a
* Content: [ASP.NET Core Blazor authentication and authorization](https://docs.microsoft.com/en-us/aspnet/core/security/blazor/?view=aspnetcore-3.1&tabs=visual-studio#authenticationstateprovider-service)
* Content Source: [aspnetcore/security/blazor/index.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/blazor/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Thanks @bdnts! I think that's a fine suggestion. :+1: Console is *OK*; but when demoing things, the framework is rather *chatty* in a console window making it hard to find the outcome of the demo! 😵

I'm going to have to push this back a bit tho ... it will be perhaps a month before I finally get to it. I'm 🏃😅 right now on high priority issues for the next couple of weeks.

**Comment by bdnts** [comment]
You're welcome.  And NP, we've all got delivery pressure.  That's why we're working on the weekend! :-( 
I've got to deliver a Blazor website with modified MS Identity (userid instead of email) before next Saturday.  There is a bit of a gap in the documentation (for the moment), so I've created BlazorIdentity repo (https://github.com/bdnts/BlazorIdentity) along with all my notes.  Everyone feel free to purloin or perfect.  This ole brain needs all the help it can get1.  :-)
